proposals/multi-value/Overview.md

```diff
@@ -71,23 +71,31 @@ An addition function returning an additional carry bit.
 Conditionally manipulating a stack operand without using a local.
 ```wasm
 (func $add64_u_saturated (param i64 i64) (result i64)
-	($i64.add_u_carry (get_local 0) (get_local 1) (i32.const 0))
+	(call $add64_u_with_carry (local.get 0) (local.get 1) (i32.const 0))
 	(if (param i64) (result i64)
 		(then (drop) (i64.const 0xffff_ffff_ffff_ffff))
 	)
 )
 ```
 
-An iterative factorial function whose loop doesn't use locals, but uses arguments like phis.
+An iterative factorial function whose loop doesn't use locals, but uses arguments like PHIs.
```

Comment:
Same here.

Reply:
yeah main thing I was hoping was to avoid someone seeing phis and thinking did that mean this? as you'll notice the example follows. Anyway hopefully new specs can try to avoid unnecessary jargon in examples.

Reply:
done